### ISSUE TITLE
12-Hour Clock?

### ISSUE THREAD

**Issue by Hyper-Snoc** [cooolgamer/Custom-Homemenu-3DS#9]
I was wondering if you could implement a 12-Hour clock into the system for regions where such clocks are common?

**Comment by cooolgamer** [comment]
Sorry but I have no reverse engine skills, and even if I did, this is pretty hard to do

**Comment by schrmh** [comment]
@Hyper-Snoc Try this: change `%H` to `%h` in the text labelled `lau_hours` within `hud_msbt.LZ`. There won't be AM/PM indicators however.
@ManiacOfGitHub found this out.

**Comment by ManiacOfGitHub** [comment]
This is true, however, I should note that it uses the Japanese 12-hour clock system, meaning that noon and midnight show as 0:00 instead of 12:00. Other than that, it works as you would expect. Also, obviously, since this is only a Home Menu edit, it won't affect any of the system applets like the eShop, System Settings, etc., although you may be able to make that change to some of those apps as well.
